src/test/app/PayChan_test.cpp

```diff
@@ -24,9 +24,9 @@
 #include <ripple/protocol/PayChan.h>
 #include <ripple/protocol/TxFlags.h>
 #include <ripple/protocol/jss.h>
-#include <test/jtx.h>
-
+#include <ripple/rpc/impl/RPCHelpers.h>
```

Comment:
Why did you change the headers order? The original order and the new line looks right to me.